Client/Frontend/Home/HomepageViewModel.swift

```diff
@@ -6,6 +6,11 @@ import MozillaAppServices
 
 protocol HomepageViewModelDelegate: AnyObject {
     func reloadSection(section: HomepageViewModelProtocol)
```

Comment:
Just to understand, when all section have their data adaptor, would we remove reloadSection(section:) and just keep reloadData()?

Reply:
Yeah this was just a work around because I don't want to disrupt how the other view models are working. We'll need another task to do clean up once all of the child view models have been updated. 
This also made me realize I need to call ensureMainThread for the actual reloadData call in the view controller. I initially thought it wouldn't be needed but since the adaptor reaches back to trigger an update it can actually come from a background thread.